packages/server/lib/util/args.js

```diff
@@ -287,6 +287,20 @@ module.exports = {
       options.config = {}
     }
 
+    const invalidConfigOptions = _.keys(options.config).reduce((invalid, option) => {
+      if (!configKeys.find((configKey) => configKey === option)) {
+        invalid.push(option)
+      }
+
+      return invalid
+    }, [])
+
+    if (invalidConfigOptions.length && !options.invokedFromCli) {
```

Comment:
Why is this ignoring `--config` invoked from the CLI? Isn't this the exact use case that the original issue was meant to address? 

I was trying to test this manually and it never warns with an error because `!options.invokedFromCli` is true.

```
cypress run --config trashAssetsBeforeHeadlessRun=false
```

Reply:
Hi Jennifer

I added the check because otherwise the warning is showed twice
<img width="687" alt="Screenshot 2020-08-11 at 09 21 15" src="https://user-images.githubusercontent.com/6031800/89869041-0e602680-dbb4-11ea-9bd7-71a40f6b03ec.png">

It was working for me, probably, because I was using a different command, I can reproduce it not working if I used the same command you used. Do you have a suggestion for preventing it from showing the warning twice?